doc/releases/reef.rst

```diff
@@ -5,6 +5,417 @@ Reef
+* rgw/kafka/amqp: fix race conditionn in async completion handlers (`pr#54736 <https://github.com/ceph/ceph/pull/54736>`_, Yuval Lifshitz)
+* rgw/lc: do not add datalog/bilog for some lc actions (`pr#55289 <https://github.com/ceph/ceph/pull/55289>`_, Juan Zhu)
+* rgw/lua: fix CopyFrom crash (`pr#54296 <https://github.com/ceph/ceph/pull/54296>`_, Yuval Lifshitz)
+* rgw/notification: Kafka persistent notifications not retried and removed even when the broker is down (`pr#56140 <https://github.com/ceph/ceph/pull/56140>`_, kchheda3)
+* rgw/putobj: RadosWriter uses part head object for multipart parts (`pr#55621 <https://github.com/ceph/ceph/pull/55621>`_, Casey Bodley)
+* rgw/rest: fix url decode of post params for iam/sts/sns (`pr#55356 <https://github.com/ceph/ceph/pull/55356>`_, Casey Bodley)
+* rgw/S3select: remove assert from csv-parser, adding updates (`pr#55969 <https://github.com/ceph/ceph/pull/55969>`_, Gal Salomon)
+* RGW/STS: when generating keys, take the trailing null character into account (`pr#54127 <https://github.com/ceph/ceph/pull/54127>`_, Oguzhan Ozmen)
+* rgw: add headers to guide cache update in 304 response (`pr#55094 <https://github.com/ceph/ceph/pull/55094>`_, Casey Bodley, Ilsoo Byun)
+* rgw: Add missing empty checks to the split string in is_string_in_set() (`pr#56347 <https://github.com/ceph/ceph/pull/56347>`_, Matt Benjamin)
+* rgw: d3n: fix valgrind reported leak related to libaio worker threads (`pr#54852 <https://github.com/ceph/ceph/pull/54852>`_, Mark Kogan)
+* rgw: do not copy olh attributes in versioning suspended bucket (`pr#55606 <https://github.com/ceph/ceph/pull/55606>`_, Juan Zhu)
+* rgw: fix cloud-sync multi-tenancy scenario (`pr#54328 <https://github.com/ceph/ceph/pull/54328>`_, Ionut Balutoiu)
+* rgw: object lock avoids 32-bit truncation of RetainUntilDate (`pr#54674 <https://github.com/ceph/ceph/pull/54674>`_, Casey Bodley)
+* rgw: only buckets with reshardable layouts need to be considered for resharding (`pr#54129 <https://github.com/ceph/ceph/pull/54129>`_, J. Eric Ivancich)
+* RGW: pubsub publish commit with etag populated (`pr#56453 <https://github.com/ceph/ceph/pull/56453>`_, Ali Masarwa)
+* rgw: RGWSI_SysObj_Cache::remove() invalidates after successful delete (`pr#55716 <https://github.com/ceph/ceph/pull/55716>`_, Casey Bodley)
+* rgw: SignatureDoesNotMatch for certain RGW Admin Ops endpoints w/v4 auth (`pr#54791 <https://github.com/ceph/ceph/pull/54791>`_, David.Hall)
+* Snapshot schedule show subvolume path (`pr#56419 <https://github.com/ceph/ceph/pull/56419>`_, Ivo Almeida)
+* src/common/options: Correct typo in rgw.yaml.in (`pr#55445 <https://github.com/ceph/ceph/pull/55445>`_, Anthony D'Atri)
+* src/mount: kernel mount command returning misleading error message (`pr#55300 <https://github.com/ceph/ceph/pull/55300>`_, Neeraj Pratap Singh)
+* src/osd/OSDMap.cc: Fix encoder to produce same bytestream (`pr#55712 <https://github.com/ceph/ceph/pull/55712>`_, Kamoltat)
```

Comment:
This went out in 18.2.2 and should be dropped from the list.

Reply:
only line 409?

Reply:
Yes to all three questions -- you can cross check against 18.2.2 release notes in the same file yourself.